src/ucp/api/ucp.h

```diff
@@ -189,6 +189,20 @@ enum ucp_listener_params_field {
 };
 
 
+/**
+ * @ingroup UCP_WORKER
+ * @brief UCP worker address flags.
+ *
+ * The enumeration list describes possible UCP worker address flags, indicating
+ * what needs to be included to the worker address returned by
+ * @ref ucp_worker_query "ucp_worker_query()" routine.
+ */
+typedef enum {
+    UCP_WORKER_ADDRESS_FLAG_NET_ONLY = UCS_BIT(0) /**< Pack addresses of network
```

Comment:
Please add justification why it is useful. You can give an example that it can be used to reduce message size for OOB address exchanges.